timemachine/maps/terminal_bonds.py

```diff
@@ -0,0 +1,239 @@
+"""Summarize HarmonicBonds as Intervals in R+, construct maps between Intervals, & apply these maps to bond lengths"""
+
+from dataclasses import dataclass
+from typing import Tuple
+
+import numpy as np
+from jax import config, jacobian, jit
+from jax import numpy as jnp
+from jax import vmap
+from numpy.typing import NDArray as Array
+
+config.update("jax_enable_x64", True)
+
+from typing import List
+
+import networkx as nx
+
+from timemachine.constants import BOLTZ, DEFAULT_TEMP
+
+
+@dataclass
+class Interval:
+    lower: float
+    upper: float
+
+    @property
+    def width(self) -> float:
+        return self.upper - self.lower
+
+    def validate(self) -> None:
+        assert self.width > 0
+        assert self.lower > 0
+
+
+@dataclass
+class Gaussian:
+    mean: float
+    stddev: float
+
+    def to_interval(self, sigma_thresh=20) -> Interval:
+        r = self.stddev * sigma_thresh
+        interval = Interval(self.mean - r, self.mean + r)
+        interval.validate()
+        return interval
+
+
+def gaussians_from_harmonic_bonds(ks, eq_lengths, temperature=DEFAULT_TEMP) -> List[Gaussian]:
+    kBT = BOLTZ * temperature
+    params = zip(ks, eq_lengths)
+    return [Gaussian(eq_length, np.sqrt(kBT / k)) for (k, eq_length) in params]
```

Comment:
nit: this function seems like it does 2 distinct things:
1. translating harmonic bond parameters to mu, sigma
2. mapping over a list of inputs

Reply:
Good point -- would be cleaner to remove this free-floating function, add a `Gaussian.from_harmonic_bond`, and do the list comprehension where needed -- will fix!

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/801/commits/ea1760e95725e4b8e97daef6bf99648ea561c79a